stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewGroupDescriptor.java

```diff
@@ -66,15 +75,30 @@ protected Object onGetChildAt(ViewGroup element, int index) {
     return context.getChildAt(index);
   }
 
-  private final class ElementContext implements ViewGroup.OnHierarchyChangeListener {
-    // This is a cache that maps from a View to the Fragment that contains it. If the
-    // View isn't contained by a Fragment, then this maps the View to itself.
-    // For Views contained by Fragments, we emit the Fragment instead, and then let
-    // the Fragment's descriptor emit the View as its sole child. This allows us to
-    // see Fragments in the inspector as part of the UI tree.
-    private final Map<View, Object> mViewToElementMap =
-        Collections.synchronizedMap(new IdentityHashMap<View, Object>());
+  private Object getElementForView(ViewGroup parentView, View view) {
```

Comment:
Hmm it made sense at one point. I'll just remove the null check. I know why I did this but you'd probably disagree :) It doesn't match up with the usage anymore (a fact which isn't affected by this change btw).